test-packages/ember-cli-fastboot-testing-app/.travis.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
btw, I think we can drop this file (as you have already added the test case to Github Actions!)

Reply:
Sure. Will remove it.